src/content/app/tools/blast/state/blast-form/blastFormSelectors.ts

```diff
@@ -23,3 +23,6 @@ export const getEmptyInputVisibility = (state: RootState) =>
   state.blast.blastForm.shouldAppendEmptyInput;
 
 export const getStep = (state: RootState) => state.blast.blastForm.step;
+
+export const getSelectedSpecies = (state: RootState) =>
```

Comment:
Since the `selectedSpecies` field at the moment only contains genome ids, it would probably be more appropriate to call this selector `getSelectedSpeciesIds`

Reply:
Done